src/main/java/io/pivotal/migration/MigrationClient.java

```diff
@@ -757,6 +765,29 @@ else if ("pending".equals(status)) {
 		}
 	}
 
+	private boolean updateStateReasonToNotPlanned(JiraIssue jiraIssue, String url) {
+		RequestEntity<String> request = RequestEntity.patch(url)
+				.accept(new MediaType("application", "vnd.github.golden-comet-preview+json"))
+				.header("Authorization", "token " + this.config.getAccessToken())
+				.body("{\n" +
+						"  \"state\": \"closed\", \n" +
+						"  \"state_reason\": \"not_planned\"\n" +
+						"}");
```

Comment:
Arent' we won Java 17 btw? Ah anyway, lets not get distracted with details and text blocks. ;)